Color/src/Graphics/Color/Space/CIE1976/LAB/LCH.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- |
+-- Module: Graphics.Color.Space.CIE1976.LAB.LCH
+
+module Graphics.Color.Space.CIE1976.LAB.LCH
+  ( pattern ColorLCH
+  , pattern ColorLCHA
+  , LCH
+  , Color(LCHab)
+  ) where
+
+import Data.Coerce
+import Data.Proxy
+import Foreign.Storable
+import qualified Graphics.Color.Model.LCH as CM
+import Graphics.Color.Space.CIE1976.LAB
+import Graphics.Color.Model.Internal
+import Graphics.Color.Space.Internal
+
+-- | [CIE L*C*Hab](https://en.wikipedia.org/wiki/CIELAB_color_space),
+--   an LCH representation for the L*a*b* color space
+data LCH (i :: k)
```

Comment:
I think it does make sense to use separate names for `LCHab` and `LCHuv` color spaces.
```suggestion
data LCHab (i :: k)
```
It will avoid name clashing and the distinction will be clear when used